fe/src/App.tsx

```diff
@@ -1,8 +1,13 @@
+import { Home } from '@Pages/Home/index';
+import { ThemeProvider } from 'styled-components';
+
 import { GlobalStyle } from '@Styles/GlobalStyle';
+import { theme } from '@Styles/theme';
```

Comment:
[제안]
내부 모듈 import와 외부 라이브러리에서의 import를 구분해 기재해두는 것을 추천드립니다~ 🤓
```suggestion
import { ThemeProvider } from 'styled-components';

import { Home } from '@Pages/Home/index';
import { GlobalStyle } from '@Styles/GlobalStyle';
import { theme } from '@Styles/theme';
```

Reply:
eslint import rule에 따라 자동 정렬되고 있어 신경 못 썼는데, 전반적으로 검토하고 Rule에도 적용 가능한지 확인하겠습니다📖